Extensions/satsukimous.js

```diff
@@ -11,8 +11,20 @@ XKit.extensions.satsukimous = new Object({
 	running: false,
 
 	preferences: {
+		replacement: {
+			text: "Replacement Avatar",
+			default: "https://31.media.tumblr.com/avatar_0bc380bccba7_128.png",
+			value: "https://31.media.tumblr.com/avatar_0bc380bccba7_128.png",
+			type: "combo",
+			values: [
+				"Satsuki Kiryūin", "https://31.media.tumblr.com/avatar_0bc380bccba7_128.png",
+				"Matoi Ryūko", "http://38.media.tumblr.com/avatar_2e71003ae267_128.png",
```

Comment:
Somehow I cannot believe that we are talking about Kill La Kill characters
in the most serious way possible.

On Tue, Aug 25, 2015, 19:01 Bryan Tsang <notifications@github.com> wrote:

> In Extensions/satsukimous.js
> <https://github.com/new-xkit/XKit/pull/350#discussion_r37890598>:
>
> > @@ -11,8 +11,20 @@ XKit.extensions.satsukimous = new Object({
> >  	running: false,
> >
> >  	preferences: {
> > +		replacement: {
> > +			text: "Replacement Avatar",
> > +			default: "https://31.media.tumblr.com/avatar_0bc380bccba7_128.png",
> > +			value: "https://31.media.tumblr.com/avatar_0bc380bccba7_128.png",
> > +			type: "combo",
> > +			values: [
> > +				"Satsuki Kiryūin", "https://31.media.tumblr.com/avatar_0bc380bccba7_128.png",
> > +				"Matoi Ryūko", "http://38.media.tumblr.com/avatar_2e71003ae267_128.png",
>
> To keep with the naming convention, this should be Ryūko Matoi.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/new-xkit/XKit/pull/350/files#r37890598>.
>